### PR TITLE
Update to run with latest Dallinger

### PR DESCRIPTION
**Pull request by alecpm**
See #242

### CODE REVIEW

**Comment by alecpm** [comment]
The `issues/242-dallinger-updates` branch of GridUniverse should now run successfully with the `issues/5330-websocket-updates` branch of Dallinger.

File: dlgr/griduniverse/experiment.py [hunk 1]
```diff
@@ -1099,6 +1092,7 @@ def fermi(beta, p1, p2):
 @extra_routes.route("/consent")
 def consent():
     """Return the consent form. Here for backwards-compatibility with 2.x."""
+    config = get_config()
```
[the remark this answers is not in the document]
Comment (alecpm) [reply dlgr/griduniverse/experiment.py:1095]: Yeah, the old code resulted in an import loop and failure.

File: dlgr/griduniverse/static/scripts/util/convert.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
-var flatten = require("flatten");
+var lodash = require("lodash");
```
[the remark this answers is not in the document]
Comment (alecpm) [reply dlgr/griduniverse/static/scripts/util/convert.js:1]: That would be nice. Running `yarn install` gave me a warning that `flatten` was dead and I should use `lodash` instead, so I did the minimum to make that change. I think generally we probably want to get this all running on NPM stable with reasonably recent releases, if possible. When we do that we can probably remove some of these micro-libraries.

**Comment by jessesnyder** [comment]
@alecpm Given that current `master` is broken, any reason this can't be merged?

**Comment by alecpm** [comment]
None that I know of. Merging.

**Pull request merged**